src/index.ts

```diff
@@ -263,6 +264,11 @@ const createWindow = () => {
     }
   });
 
+  // Inject custom CSS
+  mainWindow.webContents.on('dom-ready', () => {
+    mainWindow?.webContents.send('inject-css', loadCustomCSS());
```

Comment:
Why do we need to send a new event here? Why not do the actual work in this callback?

Reply:
Because index.ts is part of the main process and thus does not have access to the document, which only the renderer process (app.tsx) has access to, so we can't manipulate the DOM from index.ts directly

Reply:
thanks - makes sense